api/helpers/getDataWithoutToken.ts

```diff
@@ -0,0 +1,9 @@
+import { omit } from 'lodash';
```

Comment:
Is should be better to import only omit from `lodash/omit`

`import omit from 'lodash/omit';`